Wikipedia/Code/Panels.swift

```diff
@@ -0,0 +1,111 @@
+
+class EnableReadingListSyncPanelViewController : ScrollableEducationPanelViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        image = UIImage.init(named: "reading-list-syncing")
+        heading = WMFLocalizedString("reading-list-sync-enable-title", value:"Turn on reading list syncing?", comment:"Title describing reading list syncing.")
+        subheading = WMFLocalizedString("reading-list-sync-enable-subtitle", value:"Your saved articles and reading lists can now be saved to your Wikipedia account and synced across devices.", comment:"Subtitle describing reading list syncing.")
+        primaryButtonTitle = WMFLocalizedString("reading-list-sync-enable-button-title", value:"Enable syncing", comment:"Title for button enabling reading list syncing.")
+    }
+}
+
+extension UIViewController {
+    @objc func wmf_showEnableReadingListSyncPanelOnce(theme: Theme) {
+        guard !UserDefaults.wmf_userDefaults().wmf_didShowEnableReadingListSyncPanel() else {
```

Comment:
@joewalsh You had mentioned not using user defaults for these `once per user login` panels... what was the alternative again? (The way I've wired this one up is `once ever` a.t.m. - I'll will rename it when it's switched to `once per user login`.)

Reply:
In retrospect, it'ss probably fine to use defaults for these